### ISSUE TITLE
getFile fails after upgrade to v19 - backward compatibility broken

### ISSUE THREAD

**Issue by dantrevino** [stx-labs/stacks.js#624]
In my component I'm doing this:
```
import { getFile } from 'blockstack'
...
getFile(this.data.attachments[0]).then(dataUrl => {
          this.link = dataUrl
        })
```

With version 19.0.0, I'm getting the following errors in the console, and data is not loading:
```
[Vue warn]: Error in created hook: "TypeError: Cannot read property 'UserSession' of undefined"
![image](https://user-images.githubusercontent.com/7738/54254457-04467e00-4511-11e9-8a03-4619c65fd2fd.png)

**Comment by hstove** [comment]
Hey Dan - sorry to just get back to you.

We fixed this issue in a recent PR: #613. Unfortunately, we haven't published a new beta version on NPM with the fix, yet.

The workaround is to `import * as blockstack from 'blockstack'`, and then `blockstack.getFile()`. After we publish a new version, your original code will work.

**Comment by zone117x** [comment]
@hstove @yknl I think npm beta package https://www.npmjs.com/package/blockstack/v/19.0.1-beta.1 should include this fix?

**Comment by yknl** [comment]
Yes the beta for v19.0.1 includes the fix from @zone117x

**Comment by zone117x** [comment]
This should be fixed in latest v19.1.0 release. This and related functions should work with the following import methods.

```js
const blockstack = require('blockstack');
import blockstack from 'blockstack';
import * as blockstack from 'blockstack';
```
